ui/src/common/utils.js

```diff
@@ -221,7 +221,8 @@ export function addOrdinalSuffix(i) {
 }
 
 const numberAbbreviator = new NumberAbbreviator(['K', 'M', 'B', 'T']);
-export function abbreviateNumber(number) {
+export function abbreviateNumber(number, biggerThan = 999) {
+  if (number <= biggerThan) return number;
```

Comment:
please move that logic where it's needed, we can change the common function when it's needed for at least 2 usecases.

Reply:
also make sure this is change is tested.